content/docs/command-reference/gc.md

```diff
@@ -40,12 +40,13 @@ restored using `dvc fetch`, as long as they have previously been uploaded with
 
 ### Removing data in remote storage
 
-If `--cloud` option is provided, command deletes unused data not only in local
-DVC cache, but also in remote storage. It means it can be dangerous since in
-most cases removing data locally and in remote storage is irreversible.
+If the `--cloud` option is provided, this command deletes unused data from the
+[default remote storage](/doc/command-reference/remote/default) **in addition**
+to deleting it from the local DVC cache. To specify a DVC remote to delete from,
+use `--remote` as well.
```

Comment:
What happens if you use `--remote` without `--cloud`?

Reply:
Looks like the remote is ignored and it operates locally instead. A bit confusing but this text is correct ATM.

Reply:
p.s. It should probably fail or give a clearer warning at least. Want me to create an issue about it on the core repo @dberenbaum ?

Reply:
p.p.s. the warning is always the same even with `-c`:
```
WARNING: This will remove all cache except items used in the workspace of the current repo.
```
No mention of cloud/remotes.

Reply:
Okay, opened https://github.com/iterative/dvc/issues/6970